### ISSUE TITLE
Sandbox crashloop with JdbcBatchUpdateException [BUG] 

### ISSUE THREAD

**Issue by ashfaq-shaik** [digital-asset/daml#11556]
DAML SDK Sandbox version 1.11.1

Sandbox goes into a crashloop with a JdbcException. No pattern observed. Only option is to restart the sandbox process to recover

```
15:22:37.114 [sandbox-akka.actor.default-dispatcher-7] ERROR c.d.p.indexer.RecoveringIndexer - Error while running indexer, restart scheduled after 10 seconds 
org.h2.jdbc.JdbcBatchUpdateException: Unique index or primary key violation: "PUBLIC.INDEX_5 ON PUBLIC.PARTICIPANT_CONTRACTS(CREATE_KEY_HASH) VALUES 2294"; SQL statement:
merge into participant_contracts using dual
 on contract_id = ? when not matched then
 insert (contract_id, template_id, create_argument, create_argument_compression, create_ledger_effective_time, create_key_hash, create_stakeholders)
 values (?, ?, ?, ?, ?, ?, ?) [23505-200]
        at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1235)
        at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
        at anorm.BatchSql.$anonfun$execute$2(BatchSql.scala:83)
        at resource.AbstractManagedResource.$anonfun$acquireFor$1(AbstractManagedResource.scala:85)
        at scala.util.control.Exception$Catch.$anonfun$either$1(Exception.scala:252)
        at scala.util.control.Exception$Catch.apply(Exception.scala:228)
        at scala.util.control.Exception$Catch.either(Exception.scala:252)
        at resource.AbstractManagedResource.acquireFor(AbstractManagedResource.scala:85)
        at resource.ManagedResourceOperations.apply(ManagedResourceOperations.scala:31)
        at resource.ManagedResourceOperations.apply$(ManagedResourceOperations.scala:31)
        at resource.AbstractManagedResource.apply(AbstractManagedResource.scala:48)
        at resource.ManagedResourceOperations.acquireAndGet(ManagedResourceOperations.scala:29)
        at resource.ManagedResourceOperations.acquireAndGet$(ManagedResourceOperations.scala:29)
        at resource.AbstractManagedResource.acquireAndGet(AbstractManagedResource.scala:48)
        at anorm.BatchSql.execute(BatchSql.scala:83)
        at anorm.BatchSql.execute$(BatchSql.scala:81)
        at anorm.BatchSql$Copy.execute(BatchSql.scala:222)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.$anonfun$execute$1(ContractsTableH2.scala:87)
        at scala.Option.foreach(Option.scala:407)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.execute(ContractsTableH2.scala:87)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.$anonfun$writeState$5(TransactionsWriter.scala:52)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.writeState(TransactionsWriter.scala:52)
        at com.daml.platform.store.dao.JdbcLedgerDao.$anonfun$storeTransaction$2(JdbcLedgerDao.scala:502)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.$anonfun$runSQL$1(HikariJdbcConnectionProvider.scala:148)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.runSQL(HikariJdbcConnectionProvider.scala:148)
        at com.daml.platform.store.dao.DbDispatcher.$anonfun$executeSql$2(DbDispatcher.scala:55)
        at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
        at scala.util.Success.$anonfun$map$1(Try.scala:255)
        at scala.util.Success.map(Try.scala:213)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:180)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
ERROR: Exception while executing SQL query. Rolled back. (context: {submissionTime=2021-11-04T13:50:24.456Z, updateRecordTime=2021-11-04T13:50:24.465Z, applicationId=foobar, deduplicateUntil=2021-11-05T13:50:24.456Z, metric=store_ledger_entry, submitter=[ENTITYA], commandId=fe42cf9e-87b2-4fd1-84b5-4a385926c6d4, updateOffset=00000000000004170000000000000000, workflowId=, transactionId=0A2438656562633734392D343163622D336464382D396133612D303933343035336438623536, ledgerTime=2021-11-04T13:50:24.456Z})
org.h2.jdbc.JdbcBatchUpdateException: Unique index or primary key violation: "PUBLIC.INDEX_5 ON PUBLIC.PARTICIPANT_CONTRACTS(CREATE_KEY_HASH) VALUES 2294"; SQL statement:
merge into participant_contracts using dual
 on contract_id = ? when not matched then
 insert (contract_id, template_id, create_argument, create_argument_compression, create_ledger_effective_time, create_key_hash, create_stakeholders)
 values (?, ?, ?, ?, ?, ?, ?) [23505-200]
        at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1235)
        at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
        at anorm.BatchSql.$anonfun$execute$2(BatchSql.scala:83)
        at resource.AbstractManagedResource.$anonfun$acquireFor$1(AbstractManagedResource.scala:85)
        at scala.util.control.Exception$Catch.$anonfun$either$1(Exception.scala:252)
        at scala.util.control.Exception$Catch.apply(Exception.scala:228)
        at scala.util.control.Exception$Catch.either(Exception.scala:252)
        at resource.AbstractManagedResource.acquireFor(AbstractManagedResource.scala:85)
        at resource.ManagedResourceOperations.apply(ManagedResourceOperations.scala:31)
        at resource.ManagedResourceOperations.apply$(ManagedResourceOperations.scala:31)
        at resource.AbstractManagedResource.apply(AbstractManagedResource.scala:48)
        at resource.ManagedResourceOperations.acquireAndGet(ManagedResourceOperations.scala:29)
        at resource.ManagedResourceOperations.acquireAndGet$(ManagedResourceOperations.scala:29)
        at resource.AbstractManagedResource.acquireAndGet(AbstractManagedResource.scala:48)
        at anorm.BatchSql.execute(BatchSql.scala:83)
        at anorm.BatchSql.execute$(BatchSql.scala:81)
        at anorm.BatchSql$Copy.execute(BatchSql.scala:222)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.$anonfun$execute$1(ContractsTableH2.scala:87)
        at scala.Option.foreach(Option.scala:407)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.execute(ContractsTableH2.scala:87)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.$anonfun$writeState$5(TransactionsWriter.scala:52)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.writeState(TransactionsWriter.scala:52)
        at com.daml.platform.store.dao.JdbcLedgerDao.$anonfun$storeTransaction$2(JdbcLedgerDao.scala:502)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.$anonfun$runSQL$1(HikariJdbcConnectionProvider.scala:148)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.runSQL(HikariJdbcConnectionProvider.scala:148)
        at com.daml.platform.store.dao.DbDispatcher.$anonfun$executeSql$2(DbDispatcher.scala:55)
        at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
        at scala.util.Success.$anonfun$map$1(Try.scala:255)
        at scala.util.Success.map(Try.scala:213)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:180)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
15:22:47.153 [daml.index.db.threadpool.connection.indexer-0] ERROR c.d.platform.store.dao.DbDispatcher - Exception while executing SQL query. Rolled back. (context: {submissionTime=2021-11-04T13:50:24.456Z, updateRecordTime=2021-11-04T13:50:24.465Z, applicationId=foobar, deduplicateUntil=2021-11-05T13:50:24.456Z, metric=store_ledger_entry, submitter=[ENTITYA], commandId=fe42cf9e-87b2-4fd1-84b5-4a385926c6d4, updateOffset=00000000000004170000000000000000, workflowId=, transactionId=0A2438656562633734392D343163622D336464382D396133612D303933343035336438623536, ledgerTime=2021-11-04T13:50:24.456Z}) 
org.h2.jdbc.JdbcBatchUpdateException: Unique index or primary key violation: "PUBLIC.INDEX_5 ON PUBLIC.PARTICIPANT_CONTRACTS(CREATE_KEY_HASH) VALUES 2294"; SQL statement:
merge into participant_contracts using dual
 on contract_id = ? when not matched then
 insert (contract_id, template_id, create_argument, create_argument_compression, create_ledger_effective_time, create_key_hash, create_stakeholders)
 values (?, ?, ?, ?, ?, ?, ?) [23505-200]
        at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1235)
        at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
        at anorm.BatchSql.$anonfun$execute$2(BatchSql.scala:83)
        at resource.AbstractManagedResource.$anonfun$acquireFor$1(AbstractManagedResource.scala:85)
        at scala.util.control.Exception$Catch.$anonfun$either$1(Exception.scala:252)
        at scala.util.control.Exception$Catch.apply(Exception.scala:228)
        at scala.util.control.Exception$Catch.either(Exception.scala:252)
        at resource.AbstractManagedResource.acquireFor(AbstractManagedResource.scala:85)
        at resource.ManagedResourceOperations.apply(ManagedResourceOperations.scala:31)
        at resource.ManagedResourceOperations.apply$(ManagedResourceOperations.scala:31)
        at resource.AbstractManagedResource.apply(AbstractManagedResource.scala:48)
        at resource.ManagedResourceOperations.acquireAndGet(ManagedResourceOperations.scala:29)
        at resource.ManagedResourceOperations.acquireAndGet$(ManagedResourceOperations.scala:29)
        at resource.AbstractManagedResource.acquireAndGet(AbstractManagedResource.scala:48)
        at anorm.BatchSql.execute(BatchSql.scala:83)
        at anorm.BatchSql.execute$(BatchSql.scala:81)
        at anorm.BatchSql$Copy.execute(BatchSql.scala:222)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.$anonfun$execute$1(ContractsTableH2.scala:87)
        at scala.Option.foreach(Option.scala:407)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.execute(ContractsTableH2.scala:87)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.$anonfun$writeState$5(TransactionsWriter.scala:52)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.writeState(TransactionsWriter.scala:52)
        at com.daml.platform.store.dao.JdbcLedgerDao.$anonfun$storeTransaction$2(JdbcLedgerDao.scala:502)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.$anonfun$runSQL$1(HikariJdbcConnectionProvider.scala:148)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.runSQL(HikariJdbcConnectionProvider.scala:148)
        at com.daml.platform.store.dao.DbDispatcher.$anonfun$executeSql$2(DbDispatcher.scala:55)
        at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
        at scala.util.Success.$anonfun$map$1(Try.scala:255)
        at scala.util.Success.map(Try.scala:213)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:180)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
ERROR: Error while running indexer, restart scheduled after 10 seconds
org.h2.jdbc.JdbcBatchUpdateException: Unique index or primary key violation: "PUBLIC.INDEX_5 ON PUBLIC.PARTICIPANT_CONTRACTS(CREATE_KEY_HASH) VALUES 2294"; SQL statement:
merge into participant_contracts using dual
 on contract_id = ? when not matched then
 insert (contract_id, template_id, create_argument, create_argument_compression, create_ledger_effective_time, create_key_hash, create_stakeholders)
 values (?, ?, ?, ?, ?, ?, ?) [23505-200]
        at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1235)
        at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
        at anorm.BatchSql.$anonfun$execute$2(BatchSql.scala:83)
        at resource.AbstractManagedResource.$anonfun$acquireFor$1(AbstractManagedResource.scala:85)
        at scala.util.control.Exception$Catch.$anonfun$either$1(Exception.scala:252)
        at scala.util.control.Exception$Catch.apply(Exception.scala:228)
        at scala.util.control.Exception$Catch.either(Exception.scala:252)
        at resource.AbstractManagedResource.acquireFor(AbstractManagedResource.scala:85)
        at resource.ManagedResourceOperations.apply(ManagedResourceOperations.scala:31)
        at resource.ManagedResourceOperations.apply$(ManagedResourceOperations.scala:31)
        at resource.AbstractManagedResource.apply(AbstractManagedResource.scala:48)
        at resource.ManagedResourceOperations.acquireAndGet(ManagedResourceOperations.scala:29)
        at resource.ManagedResourceOperations.acquireAndGet$(ManagedResourceOperations.scala:29)
        at resource.AbstractManagedResource.acquireAndGet(AbstractManagedResource.scala:48)
        at anorm.BatchSql.execute(BatchSql.scala:83)
        at anorm.BatchSql.execute$(BatchSql.scala:81)
        at anorm.BatchSql$Copy.execute(BatchSql.scala:222)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.$anonfun$execute$1(ContractsTableH2.scala:87)
        at scala.Option.foreach(Option.scala:407)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.execute(ContractsTableH2.scala:87)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.$anonfun$writeState$5(TransactionsWriter.scala:52)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.writeState(TransactionsWriter.scala:52)
        at com.daml.platform.store.dao.JdbcLedgerDao.$anonfun$storeTransaction$2(JdbcLedgerDao.scala:502)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.$anonfun$runSQL$1(HikariJdbcConnectionProvider.scala:148)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.runSQL(HikariJdbcConnectionProvider.scala:148)
        at com.daml.platform.store.dao.DbDispatcher.$anonfun$executeSql$2(DbDispatcher.scala:55)
        at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
        at scala.util.Success.$anonfun$map$1(Try.scala:255)
        at scala.util.Success.map(Try.scala:213)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:180)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
15:22:47.155 [sandbox-akka.actor.default-dispatcher-6] ERROR c.d.p.indexer.RecoveringIndexer - Error while running indexer, restart scheduled after 10 seconds 
org.h2.jdbc.JdbcBatchUpdateException: Unique index or primary key violation: "PUBLIC.INDEX_5 ON PUBLIC.PARTICIPANT_CONTRACTS(CREATE_KEY_HASH) VALUES 2294"; SQL statement:
merge into participant_contracts using dual
 on contract_id = ? when not matched then
 insert (contract_id, template_id, create_argument, create_argument_compression, create_ledger_effective_time, create_key_hash, create_stakeholders)
 values (?, ?, ?, ?, ?, ?, ?) [23505-200]
        at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1235)
        at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
        at anorm.BatchSql.$anonfun$execute$2(BatchSql.scala:83)
        at resource.AbstractManagedResource.$anonfun$acquireFor$1(AbstractManagedResource.scala:85)
        at scala.util.control.Exception$Catch.$anonfun$either$1(Exception.scala:252)
        at scala.util.control.Exception$Catch.apply(Exception.scala:228)
        at scala.util.control.Exception$Catch.either(Exception.scala:252)
        at resource.AbstractManagedResource.acquireFor(AbstractManagedResource.scala:85)
        at resource.ManagedResourceOperations.apply(ManagedResourceOperations.scala:31)
        at resource.ManagedResourceOperations.apply$(ManagedResourceOperations.scala:31)
        at resource.AbstractManagedResource.apply(AbstractManagedResource.scala:48)
        at resource.ManagedResourceOperations.acquireAndGet(ManagedResourceOperations.scala:29)
        at resource.ManagedResourceOperations.acquireAndGet$(ManagedResourceOperations.scala:29)
        at resource.AbstractManagedResource.acquireAndGet(AbstractManagedResource.scala:48)
        at anorm.BatchSql.execute(BatchSql.scala:83)
        at anorm.BatchSql.execute$(BatchSql.scala:81)
        at anorm.BatchSql$Copy.execute(BatchSql.scala:222)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.$anonfun$execute$1(ContractsTableH2.scala:87)
        at scala.Option.foreach(Option.scala:407)
        at com.daml.platform.store.dao.events.ContractsTableH2$InsertContractsExecutable.execute(ContractsTableH2.scala:87)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.$anonfun$writeState$5(TransactionsWriter.scala:52)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.events.TransactionsWriter$PreparedInsert.writeState(TransactionsWriter.scala:52)
        at com.daml.platform.store.dao.JdbcLedgerDao.$anonfun$storeTransaction$2(JdbcLedgerDao.scala:502)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.$anonfun$runSQL$1(HikariJdbcConnectionProvider.scala:148)
        at com.daml.metrics.Timed$.$anonfun$value$1(Timed.scala:19)
        at com.codahale.metrics.Timer.time(Timer.java:118)
        at com.daml.metrics.Timed$.value(Timed.scala:19)
        at com.daml.platform.store.dao.HikariJdbcConnectionProvider.runSQL(HikariJdbcConnectionProvider.scala:148)
        at com.daml.platform.store.dao.DbDispatcher.$anonfun$executeSql$2(DbDispatcher.scala:55)
        at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
        at scala.util.Success.$anonfun$map$1(Try.scala:255)
        at scala.util.Success.map(Try.scala:213)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:180)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by gerolf-da** [comment]
Hi @ashfaq-shaik, I see that you reported this for SDK 1.11.1. The issue you're describing was fixed in SDK 1.11.2. Could you try with this latest bugfix version and see if you can still reproduce the problem?

**Comment by ashfaq-shaik** [comment]
Will try upgrading to SDK 1.11.2
